src/Sulu/Bundle/MediaBundle/Entity/CollectionRepository.php

```diff
@@ -295,12 +293,10 @@ private function getIdsQuery(
     ) {
         $queryBuilder = $this->createQueryBuilder('collection')
             ->select($select)
-            ->leftJoin('collection.children', 'collectionChildren')
-            ->where('collection.depth <= :depth1 OR collectionChildren.depth <= :depth2');
```

Comment:
@alexander-schranz Do you remember what this code was covering? We are experiencing problems because of it, is it still needed? Why do we have to load the children of a collection?